tests/Oryx.Integration.Tests/PythonEndToEndTests.cs

```diff
@@ -3,61 +3,81 @@
 // Licensed under the MIT license.
 // --------------------------------------------------------------------------------------------
 
+using Microsoft.Oryx.BuildScriptGenerator.Python;
```

Comment:
Seems like my VS settings messed the order here (should have been System usings before Microsoft ones). Will fix it.